samigo/samigo-hibernate/src/java/org/sakaiproject/tool/assessment/data/dao/grading/MediaData.hbm.xml

```diff
@@ -13,7 +13,7 @@
     <many-to-one name="itemGradingData"
       class="org.sakaiproject.tool.assessment.data.dao.grading.ItemGradingData"
       column="ITEMGRADINGID" index="SAM_MEDIA_ITEMGRADING_I" />
-    <property name="media">
+    <property name="dbMedia">
```

Comment:
understood... My concern is that someone looking at this is not going to understand.
Also need some sql conversion scripts.